module/chunks/chunkVerifier.go

```diff
@@ -198,6 +206,24 @@ func (fcv *ChunkVerifier) verifyTransactionsInContext(context fvm.Context, chunk
 		return nil, nil, fmt.Errorf("cannot calculate events collection hash: %w", err)
 	}
 	if chunk.EventCollection != eventsHash {
+
+		for i, event := range events {
+
+			fcv.logger.Warn().Int("list_index", i).
+				Str("event_id", event.ID().String()).
+				Hex("event_fingerptint", event.Fingerprint()).
+				Str("event_type", string(event.Type)).
+				Str("event_tx_id", event.TransactionID.String()).
+				Uint32("event_tx_index", event.TransactionIndex).
+				Uint32("event_index", event.EventIndex).
+				Bytes("event_payload", event.Payload).
+				Str("block_id", chunk.BlockID.String()).
+				Str("collection_id", chunkDataPack.CollectionID.String()).
+				Str("result_id", result.ID().String()).
+				Uint64("chunk_index", chunk.Index).
+				Msg("not matching events debug")
```

Comment:
not sure I follow what this error message means :( `not matching events debug`.

Reply:
That's from #1058 - it was to add a log when the VN produces a different event hash than the EN